### PR TITLE
switch_to_containers: a couple of fixes

### PR DESCRIPTION
**Pull request by guits**
rename switch_to_containers scenario

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

**Comment by guits** [comment]
jenkins test dev-centos-non_container-switch_to_containers

**Comment by guits** [comment]
jenkins test dev-centos-non_container-switch_to_containers

**Comment by guits** [comment]
jenkins test dev-centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test dev-centos-non_container-switch_to_containers

**Comment by guits** [comment]
jenkins test dev-centos-non_container-switch_to_containers

**Comment by guits** [comment]
jenkins test dev-centos-non_container-switch_to_containers

File: roles/ceph-container-common/tasks/pre_requisites/prerequisites.yml [hunk 1]
```diff
@@ -88,4 +88,13 @@
...
     - ansible_os_family == 'RedHat'
-    - ansible_distribution_major_version == '8'
\ No newline at end of file
+    - ansible_distribution_major_version == '8'
+
+- name: ensure tmpfiles.d is present
+  lineinfile:
+    path: /usr/lib/tmpfiles.d/ceph-common.conf
+    line: "d /run/ceph 0770 ceph ceph -"
+    owner: root
```
Comment (leseb) [inline roles/ceph-container-common/tasks/pre_requisites/prerequisites.yml]: Why root?

Comment (guits) [reply roles/ceph-container-common/tasks/pre_requisites/prerequisites.yml]: I just took the same ownership than what is done by the packaging actually

Comment (guits) [reply roles/ceph-container-common/tasks/pre_requisites/prerequisites.yml]: (It shouldn't be an issue since it's 644 and only read by systemd?)

**Comment by guits** [comment]
unrelated failure

**Pull request merged**